public/content/lessons/2017/eulers-formula-via-group-theory/index.mdx

```diff
@@ -1,9 +1,215 @@
+
+But expressions like $2^{1/2}$ or $2^{-1}$, and much less $2^i$, don’t really make sense if you think of exponents as repeated multiplication. What does it mean to multiply 2 by itself half of a time, or $-1$ of a time?
+
+So we do something that’s very common throughout math, and extend the definition to apply to exponents beyond counting numbers. But we don’t just do this randomly, if you think back to how fractional and negative exponents are defined, it’s always motivated by trying to make sure this property $2^{x+y}$ = $2^x 2^y$ still holds.
+
+To see what this might mean for complex exponents, think of what this property is saying from a group theory perspective. It’s saying that adding the inputs corresponds with multiplying the outputs, and that makes it very tempting to think of the inputs not merely as numbers, but as members of the additive group of sliding actions, and to think of the outputs not merely as numbers, but as members of the multiplicative group of stretching and squishing actions.
+
+<Figure image="add_multiply_homo.png" width="60%" />
+
+It’s weird and strange to think about a function that takes in one kind of action and spits out another kind of action, but that’s actually something which comes up all the time in group theory. This exponential property is very important for this association between groups, it forms a map between the additive and multiplicative group for real numbers.
+
+This exponentiation function guarantees that if I compose two sliding actions, maybe a slide by $-1$, then a slide by $2$, it corresponds to composing the output actions, in this case squishing by $2^{-1}$, then stretching by $2^2$.
+$$
+\begin{align*}
+2^{-1+2}&=2^{-1}\cdot2^2 \\
+\exp(-1+2)&=\exp(-1)\cdot\exp(2)
+\end{align*}

+$$
+Mathematicians would describe a property like this by saying the function “preserves the group structure.” In this sense, the arithmetic within a group is what gives it its structure, and function like this exponential plays nicely with that arithmetic. Functions between groups that preserve the arithmetic like this are really important throughout group theory, enough so that they’ve earned themselves a fancy name: “homomorphisms.”
+
+<Figure image="homomorphism.png" width="60%" />
+
+## Complex Exponentiation
+
+Now think about what this means for associating the additive group in the complex plane with the multiplicative group in the complex plane. We already know that when you plug in a real number to $2^x$, you get out a real number, a positive real number in fact, so this exponential function takes any pure horizontal slide and turns it into some pure stretching or squishing action.
+
+<Figure image="real_exponent.png" video="real_exponent.mp4" />
+
+So it would be reasonable for this new dimension of additive actions, slides up and down, to map to this new dimension of multiplicative actions, pure rotations.

```

Comment:
I think this statement raises the biggest "why though?" for me as a reader. One suggestion might be:

"As we discussed above, there are two actions we identified with complex multiplication, a stretching action and a rotating action. Since the real component of a complex number already maps to the stretching action, it would be reasonable for this new dimension, the complex component that slides up and down, to map to the multiplicative action of rotation."

Reply:
Also a footnote discussing the guts of complex multiplication and the invention/discovery of the imaginary number i could be helpful here. I know this is explored in the lockdown math series, but I don't know if those videos are slated to be converted.